content/develop/reference/modules/modules-api-ref.md

```diff
@@ -4489,7 +4489,7 @@ latency-monitor-threshold.
 ## Blocking clients from modules
 
 For a guide about blocking commands in modules, see
-[/develop/reference/modules/modules-blocking-ops]({{< relref "/develop/reference/modules/modules-blocking-ops" >}}).
+[https://redis.io/topics/modules-blocking-ops](https://redis.io/topics/modules-blocking-ops).
```

Comment:
Dodgy link again.

Reply:
Same issue: these links are embedded in source code, so I'll need to hunt them down and fix them there.

Fixed with next push.